docs/source/getting-started/authenticators-users-basics.md

```diff
@@ -81,7 +83,7 @@ the ability to manage users on the local system. When you try to add a
 new user to the Hub, a `LocalAuthenticator` will check if the user
 already exists. If you set the configuration value, `create_system_users`,
 to `True` in the configuration file, the `LocalAuthenticator` has
-the privileges to add users to the system. The setting in the config
+the privilege to add users to the system. The setting in the config
```

Comment:
```suggestion
the ability to add users to the system. The setting in the config
```
I think this is a good chance to use a different term like `ability` here, since we're talking about the functionality of the `LocalAuthenticator`. To me `privilege` implies something in the JupyterHub permissions system.